packages/cyberstorm/src/components/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropsWithChildren, ReactNode } from "react";
+import * as RadixTooltip from "@radix-ui/react-tooltip";
+import styles from "./Tooltip.module.css";
+
+export interface TooltipProps {
+  content?: ReactNode;
```

Comment:
if (and only if) the content is always a string, change the prop's type to string. If the content can be something else (as I think it can) maybe you could add a story to showcase how that looks/works. (For example pass something like a Title, TextInput and a Button wrapped in a div).